core/templates/dev/head/library/library.html

```diff
@@ -56,16 +56,16 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
                translate="I18N_LIBRARY_NO_EXPLORATION_GROUPS">
           </div>
 
-          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
+          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index" ng-init="groupIndex = $index">
```

Comment:
Drop the ng-init, but use ng-class and ng-show/ng-if as needed. For example, you could do something like `<span ng-if="currentlyHighlightedGroupIndex === $index"> &gt;</span>`.